thrimbletrimmer/scripts/IO.js

```diff
@@ -247,20 +248,32 @@ thrimbletrimmerSubmit = function(state) {
         video_channel:document.getElementById("StreamName").value,
         video_quality:document.getElementById('qualityLevel').options[document.getElementById('qualityLevel').options.selectedIndex].value,
         uploader_whitelist:(document.getElementById('uploaderWhitelist').value ? document.getElementById('uploaderWhitelist').value.split(','):null),
-        state:state,
+		state:state,
+		//pass back the sheet columns to check if any have changed
+		sheet_name:document.data.sheet_name,
+		event_start:document.data.event_start,
+		event_end:document.data.event_end,
+		category:document.data.category,
+		description:document.data.description,
+		notes:document.data.notes,
+
     };
     if (!!user) {
         wubData.token = user.getAuthResponse().id_token
     }
+	if (override_changes) {
+		wubData["override_changes"] = true;
+	}
     console.log(wubData);
     console.log(JSON.stringify(wubData));
 
 	if (!wubData.video_start) {alert("No start time set"); return;}
 	if (!wubData.video_end) {alert("No end time set"); return;}
 
     //Submit to thrimshim
-    var rowId = /id=(.*)(?:&|$)/.exec(document.location.search)[1];
-    fetch("/thrimshim/"+rowId, {
+	var rowId = /id=(.*)(?:&|$)/.exec(document.location.search)[1];
+	path = "/thrimshim/" + rowId + "?override_changes=" + override_changes; 
```

Comment:
it looks like we're putting override_changes in both the query param and the body. It should be one or the other.